.gitignore

```diff
@@ -5,3 +5,5 @@ Makefile
 cmake_install.cmake
 install_manifest.txt
 CTestTestfile.cmake
+
```

Comment:
Spurious line?

Reply:
This was meant to have some sort of order in `.gitignore` (like separating compilation related stuff from others)

Reply:
Ah, ok!